examples/BuddyLlama/test-llama2.py

```diff
@@ -0,0 +1,28 @@
+import torch
+from transformers import LlamaForCausalLM, LlamaTokenizer
+import torch._dynamo as dynamo
+from torch._inductor.decomposition import decompositions as inductor_decomp
+from torch._functorch.aot_autograd import aot_autograd_decompositions
+import numpy
+import os
+
+from buddy.compiler.frontend import DynamoCompiler
+from buddy.compiler.ops import tosa
+
+tokenizer = LlamaTokenizer.from_pretrained('/llama-2-7B-hf')
```

Comment:
The `/llama-2-7B-hf` seems to cause an error: 
```
huggingface_hub.utils._validators.HFValidationError: 
Repo id must use alphanumeric chars or '-', '_', '.', '--' and '..' are forbidden, 
'-' and '.' cannot start or end the name, max length is 96: '/llama-2-7B-hf'.
```
Did I miss something important, or should we remove the `/`?

Reply:
Ah, I recalled the details about the configurations here. We should modify the path to the huggingface version of the llama model, right?

Reply:
Yes

Reply:
I will change it from '/llama-2-7B-hf' to 'path to huggingface llama2 model'